app/script/backup/BackupRepository.js

```diff
@@ -150,7 +168,11 @@ z.backup.BackupRepository = class BackupRepository {
 
         return zip;
       })
-      .catch(() => {
+      .catch(error => {
+        console.error(error);
```

Comment:
`console`.. if we keep it until staging.. let's at least have a description text `console.error('foo bar', error)`